decidim-system/app/controllers/decidim/system/organizations_controller.rb

```diff
@@ -60,6 +60,17 @@ def update
 
       private
 
+      def default_params
+        {
+          host: request.host,
+          organization_admin_name: current_admin.email.split("@")[0],
+          organization_admin_email: current_admin.email,
+          available_locales: I18n.available_locales.map(&:to_s),
+          default_locale: I18n.available_locales.first.to_s,
```

Comment:
Maybe it would be better like this ?
```suggestion
          available_locales: Decidim.available_locales.map(&:to_s),
          default_locale: Decidim.default_locale.first.to_s,
```